frontend/src/app/About/About.module.scss

```diff
@@ -0,0 +1,315 @@
+//undefined stuff that was in main branch
+$bt-navbar-height: 58px;
+$bt-light-text: #8A8A8A;
+$bt-base-text: #383838;
```

Comment:
We have Tailwind colors defined as CSS variables in the `src/main.scss` file.

You can reference CSS variables like so:
```css
color: var(--slate-500);
```